docs/tutorials/basic_commands.md

```diff
@@ -248,21 +245,25 @@ $ sudo pro detach
 ```
 
 This command will disable all of the Ubuntu Pro services on the machine for you and
-get rid of the subscription stored on your machine during attach.
+remove the subscription stored on your machine during attach.
 
 ```{note}
 The detach command will also not uninstall any packages that were installed by
 any service enabled through `pro`.
 ```
 
-### Final thoughts
+### Final steps
```

Comment:
I hope you'll forgive me for the change, but I thought that the step of tearing down the VM should be presented before discussing the next steps available to the user.